main.go

```diff
@@ -15,11 +15,11 @@ const (
 )
 
 type Parcel struct {
-	Number    int
-	Client    int
-	Status    string
-	Address   string
-	CreatedAt string
+	Number     int
+	Client     int
+	Status     string
+	Address    string
+	Created_at string
```

Comment:
В го принят PascalCase
```suggestion
	CreatedAt string
```